.github/renovate.json5

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval",
+        ":semanticCommitScopeDisabled"
+    ],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "7 days",
+
+    "packageRules": [
+        // Enable major version updates, but don't auto-merge them
+        {
+            "matchUpdateTypes": ["major"]
+        },
```

Comment:
Sorry, not sure how to parse the last bit - are you saying that if we don't include this line, ...

* Option A: major version PRs will still be sent? And that if Renovate is taking too long, we can manually trigger them?
* Option B: because `:dependencyDashboardApproval` is enabled, we'll have to manually trigger PRs from major versions from the dashboard?

https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval makes me think it's the latter (B). I personally have prefer the former (A) but have gotten the impression in this PR that you all have recommended B.

Reply:
He's saying that a package rule with only `match` config does nothing. Think of a package rule as like "when you match X, do Y". You don't have any "do Y" in your rule, so it does nothing.

Your config sets dependencyDashboardApproval=true which means _no_ PR is created by default, and instead PRs are only created when someone with repo permissions checks a box in the Dependency Dashboard issue. In other words it's like "on demand" PRs instead of fully automated. It's not just major versions either, unless you write further rules to exclude non-major.

My recommendation in most cases is to use dependency dashboard approval for major updates, but allow non-major updates to be created automatically.

Reply:
Swell, thanks - I'll remove this line.